### PR TITLE
Document manual process for emailing tenants

### PR DESCRIPTION
**Pull request by benhyland**
We've been doing this for a while, it's time we wrote it down.

This process will hopefully be superseded fairly soon, but now
we have a home for docs as the process is iterated.

### CODE REVIEW

File: docs/team/notifying_tenants.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  cf target -o $org >/dev/null;
+  for space in `cf spaces 2>&1 | tail -n+4`; do
+    cf space-users $org $space;
+  done;
+done | grep '@' | sort | uniq
+```
+
+## Send the email
+
+We send the email from any team member's `gov.uk` GMail account.
```
Comment (mtekel) [inline docs/team/notifying_tenants.md]: +We use support email address as a non-alias to send from. To configure that for your account, follow two links below

**Comment by timmow** [comment]
Can we link to this from [this page](https://government-paas-team-manual.readthedocs.io/en/latest/guides/common_support_tasks/)?

File: docs/team/notifying_tenants.md (hunk 1 above)
Comment (benhyland) [inline docs/team/notifying_tenants.md]: done

**Comment by benhyland** [comment]
@timmow yep

**Comment by mtekel** [comment]
I would add a note to use bcc for the list of recipients, so that tenants can't see who else has account as well...

**Comment by benhyland** [comment]
@mtekel if you look closely, you should find this note already exists. Please feel free to add a commit making it more obvious!

**Comment by dcarley** [comment]
> @mtekel if you look closely, you should find this note already exists. Please feel free to add a commit making it more obvious!

I overlooked it the first time too. Not sure if that's because I was viewing the raw instead of rendered Markdown. It wouldn't hurt to put it as the first paragraph in the section though.

**Comment by mtekel** [comment]
hm, thanks for notifying, but I had to use search to find it :)

**Comment by benhyland** [comment]
@mtekel @dcarley Please, if you can do better, don't hesitate

**Comment by mtekel** [comment]
Before I merge, I want to get OK for the last change I did...

**Comment by benhyland** [comment]
@mtekel lgtm thanks!

**Pull request merged**